Samples/HelloWorld.NetCore/NuGet.Config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we could and should remove this file. This sample do not uses the myget feed, and if for any reason we delete this feed from myget, it will break the build.

Reply:
hmm maybe sometimes it does, like when we want to update the sample with things published on myget? lol I'm fine with both, just assume myget is configured for a reason. But I don't know much as myget. @sergeybykov  is only nightly build published in myget? or it is the one we are going to remove?

Reply:
I think sample should point to nuget.org. MyGet is only for nightly builds and bleeding edge enthusiasts.